inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterService.java

```diff
@@ -29,13 +29,22 @@
 import org.apache.inlong.manager.common.pojo.cluster.ClusterTagRequest;
 import org.apache.inlong.manager.common.pojo.cluster.ClusterTagResponse;
 import org.apache.inlong.manager.common.pojo.dataproxy.DataProxyNodeInfo;
+import org.apache.inlong.manager.dao.entity.UserEntity;
 
 import java.util.List;
 
 /**
  * Inlong cluster service layer interface
  */
 public interface InlongClusterService {
+ 
+    /**
+     * Get user info by user name
+     *
+     * @param username username
+     * @return user info
+     */
+    UserEntity getByName(String username);
```

Comment:
1. `getByUsername` is better.
2. This method should move to `UserService`, which is more object-oriented.